config/settings.yml

```diff
@@ -5,6 +5,8 @@ chef:
 doorkeeper:
   administrators:
     - rainbowdash
+email_from_address: 'Chef Notifications <notifications@chef.io>'
```

Comment:
See https://github.com/railsconfig/rails_config#common-config-file to see how the settings files are read. config/settings.yml are the base defaults, which are usually appropriate for out-of-the-box development and we usually drop a config/settings/production.yml on the box when deploying with Chef.